.babelrc

```diff
@@ -7,10 +7,12 @@
 					"last 2 Firefox versions",
 					"last 2 Safari versions",
 					"last 2 Edge versions",
+					"last 2 Opera versions",
 					"last 2 iOS versions",
 					"last 1 Android version",
 					"last 1 ChromeAndroid version",
-					"ie 11"
+					"ie 11",
+					"> 1%"
```

Comment:
Yes, correct, these are the same settings that will be merged for core in 4.9

https://core.trac.wordpress.org/ticket/37651